docs/vector_store.ipynb

```diff
@@ -423,6 +423,26 @@
     "print(docs)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initialize Vector Store with documents from any source\n"
```

Comment:
This doesn't make sense just added to the end of this section. It may be best bundled with "Initialize a default AlloyDBVectorStore" with a comment on the difference. The goal is to make it easier and more clear for users that are use to looking at docs like https://python.langchain.com/v0.1/docs/integrations/vectorstores/pinecone/

Reply:
Sure, moved it.